editor/src/components/canvas/canvas-strategies/strategies/absolute-reparent-strategy.spec.browser2.tsx

```diff
@@ -166,7 +176,184 @@ export var ${BakedInStoryboardVariableName} = (
 )`
 }
 
+function getElementByDataUID(renderResult: EditorRenderResult, dataUID: string): HTMLElement {
+  const queryByDataUID = queryHelpers.queryByAttribute.bind(null, 'data-uid')
+  return forceNotNull(
+    `Could not find element with ${dataUID}`,
+    queryByDataUID(renderResult.renderedDOM.container, dataUID),
+  )
+}
```

Comment:
Very nice! I can see this being pulled out into one of our `.test-utils` files in the very near future